docs/FFI.md

```diff
@@ -35,6 +35,7 @@ They tend to be suffixed with a `#` and require the use of `MagicHash` language
 | `Float#` | float | |
 | `Double#` | double | |
 | `JChar#` | char | |
+| `JString#`| string| To cast it to Haskell `String`, use `unpackCString` from `GHC.Pack`|
```

Comment:
Change `string` to `java.lang.String` (there's no `string` in Java like there is in C++)